src/Cell.tsx

```diff
@@ -127,6 +127,6 @@ function Cell<R, SR>(
   );
 }
 
-export default memo(forwardRef(Cell)) as <R, SR = unknown>(
+export default memo(forwardRef(Cell)) as <R, SR>(
```

Comment:
I don't think we need a default here: we need to pass a `column` prop, which will allow TS to infer the `SR` from it, so it should be fine.

![image](https://user-images.githubusercontent.com/567105/119145182-4f3d7380-ba41-11eb-8bcb-c4680d760573.png)

This also applies to other components in this PR like `Row` and `TextEditor`.